packages/fiori/src/themes/Page.css

```diff
@@ -0,0 +1,90 @@
+
+:host(:not([hidden])) {
+    width: 100%;
+    height: 100%;
+}
+
+.ui5-page-root {
+    height: 100%;
+    overflow: hidden;
+    position: relative;
+    z-index: 0;
+    box-sizing: border-box;
+}
+
+.ui5-page-header-root {
+    z-index: 1;
+}
+
+.ui5-page-content-root {
+    overflow: hidden auto;
+    position: absolute;
+    will-change: scroll-position;
+    width: 100%;
+    top: 2.75rem;
+    bottom: 0;
+    box-sizing: border-box;
+}
+
+.ui5-page-footer-root {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    z-index: 2;
+    width: 100%;
+}
+
+:host([floating-footer]) .ui5-page-footer-root {
+    left: 0.5rem;
+    right: 0.5rem;
+    width: auto;
+    opacity: 1;
+    bottom: 0.5rem;
+}
+
+:host([floating-footer]:not([hide-footer])) .ui5-page-footer-root {
+    animation: bounceShow 0.35s forwards ease-in-out;
+}
+
+:host([floating-footer][hide-footer]) .ui5-page-footer-root {
+    animation: bounceHide 0.35s forwards ease-in-out;
+}
+
+/* BackgroundDesign */
+:host([backgroundDesign="Standard"]) .ui5-page-root {
+	background-color: var(--sapBackgroundColor);
```

Comment:
Is it possible to change the content background easily (as with the CSS below).

If you set these backgrounds on the host and inherit the background in .ui5-page-root element, the users would be able to easily set a custom background like:
```css
ui5-page {
background: red;
}
```

Reply:
Thanks, done.